app/forms/support_interface/conditions_form.rb

```diff
@@ -247,5 +252,41 @@ def structured_conditions_to_save
         )
       end
     end
+
+    def ske_conditions_are_valid
+      validate_ske_conditions
+      validate_language_count if language_course?
+      validate_combined_ske_length if multiple_ske_conditions?
+    end
+
+    def validate_combined_ske_length
+      ske_conditions_to_validate = structured_conditions_to_save
+      if ske_conditions_to_validate.many? && ske_conditions_to_validate.none? { |sc| sc.length == '8' }
```

Comment:
could `ske_conditions_to_validate.none? { |sc| sc.length == '8' }` be extracted to `no_conditions_meet_min_length_criteria` or some such?

Reply:
perhaps this logic (if multiple conditions, one must be length 8 weeks) should exist a bit closer to the model

Reply:
Extracted to `SkeCondition.no_conditions_meet_minimum_length_criteria?` and removed duplication in `OfferWizard`. (Can't just put it as a validation on `Offer` because the conditions being validated haven't been added to the `Offer` yet).